sql/core/src/main/scala/org/apache/spark/sql/execution/BaseScriptTransformationExec.scala

```diff
@@ -262,7 +262,8 @@ trait BaseScriptTransformationExec extends UnaryExecNode {
       }
 }
 
-abstract class BaseScriptTransformationWriterThread extends Thread with Logging {
+abstract class BaseScriptTransformationWriterThread
+  extends Thread("Thread-ScriptTransformation-Feed") with Logging {
```

Comment:
Looks fine but can we check if this naming is consistent with others?

Reply:
We can follow a format like this.
https://github.com/apache/spark/blob/8acce8885964fcb5b23046bdac8a882d723f35cc/sql/core/src/main/scala/org/apache/spark/sql/execution/BaseScriptTransformationExec.scala#L100

![image](https://user-images.githubusercontent.com/3898450/164274148-e3097a81-4a76-4d24-85f3-0b8ec5822f81.png)